src/samples/aspnet/Elsa.Samples.ExtendHttpEndpoint/Elsa.Samples.ExtendHttpEndpoint.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <Content Remove="workflow-definition-custom-http-endpoint.json" />
+    </ItemGroup>
```

Comment:
Can we remove this?

Reply:
@sfmskywalker, workflow-definition-custom-http-endpoint.json is an designer export of the simple sample workflow using my "custom" HTTP Endpoint derived activity.
I didn't know where to put it so I placed in the folder of the sample and excluded it from C# project (hence the entry in .csproj).
Do you want me to include it in the project (because that will happen if I remove these 3 lines which exclude it)?
Or you want me to complete remove this json file from the git?

Reply:
@sfmskywalker I've removed this json file completely from git and attaching to this PR, this is simplest solution (I wanted to clear this up this week :)).